### DISCUSSION TITLE
How to write default values to `.projenrc.ts` file?

### DISCUSSION THREAD

**Discussion by shinebayar-g** [projen/projen#3797] [Q&A] [answered]
I created a custom project extended from the base `Project` class. Then I used `ProjenrcTs` component to generate a `.projenrc.ts` file.

```ts
import { Project } from 'projen';
import { ProjenrcTs } from 'projen/lib/typescript';

export interface MyProjectOptions {
    readonly name: string;
    readonly libVersion?: string;
}

export class MyProject extends Project {
    constructor(options: MyProjectOptions) {
        super(options);
        new ProjenrcTs(this);
    }
}
```

If I bootstrap a new project using `npx projen new --from "my-package" --name test` , then the `.projenrc.ts` file is generated with the following.

```ts
import { MyProject } from 'my-package';
const project = new MyProject({
    name: 'my-repo',
});
project.synth();
```

How can I generate `.projenrc.ts` file with the default values like this 

```ts
import { MyProject } from 'my-package';
const project = new MyProject({
    name: 'my-repo',
    libVersion: '1.2.3'
});
project.synth();
```

without using the `npx projen new --from "my-package" --name test --libVersion 1.2.3` command? `ProjenrcTs` component doesn't accept values.

**Comment by mrgrain** [comment, marked as answer]
Add a `@pjnew` tag to the property in your interface. It's not well-documented at the moment. Sorry about that.

Like this:
https://github.com/projen/projen/blob/d399cd89f2bf2cf050f5430827171e376d908f54/src/cdk/jsii-project.ts#L132

**Reply by shinebayar-g** [reply]
Ah gotcha. That solved my problem. Thank you.
